src/components/Asset/AssetActions/Download.tsx

```diff
@@ -267,7 +267,8 @@ export default function Download({
       </div>
 
       {asset?.accessDetails?.datatoken?.name !== '' &&
-        asset?.nft.owner === accountId && (
+        asset?.nft.owner === accountId &&
+        asset?.accessDetails?.datatoken?.symbol && (
```

Comment:
@kremalicious @claudiaHash here I'm checking `asset?.accessDetails?.datatoken?.symbol` is valued otherwise it won't render the component (this avoids breaking the component as it will never missed this .symbol). This should be enough to avoid rendering the component without it necessary values. 

I can add these same optional chaining operator in line 239, but it will be useless as it'll never reach the component without having `.symbol` valued. I can change this implementation to add the optional chain op `asset?.accessDetails?.datatoken?.symbol` directly to the component as it won't change much, but i think it's safer to avoid rendering the component without this value as if the value is `underfined` (because of the optional chaining), then the component will show "collect 0 undefined". We can have a fallback for this, but i don't think matters as it should always render the token's symbol. Does make sense or should we stick with the optional chaining as param on the component?